radiation/python/radlw/finalloop_gt4py.py

```diff
@@ -0,0 +1,144 @@
+import gt4py
+import os
+import sys
+import time
+import numpy as np
+import xarray as xr
+from gt4py.gtscript import stencil, computation, interval, PARALLEL, FORWARD
+
+sys.path.insert(0, "/Users/AndrewP/Documents/work/physics_standalone/radiation/python")
+from util import (
+    view_gt4py_storage,
+    compare_data,
+    create_storage_from_array,
+    create_storage_zeros,
+    create_storage_ones,
+    loadlookupdata,
+)
+from config import *
+
+SERIALBOX_DIR = "/Users/AndrewP/Documents/code/serialbox2/install"
+sys.path.append(SERIALBOX_DIR + "/python")
+import serialbox as ser
+
+ddir = "/Users/AndrewP/Documents/work/physics_standalone/radiation/fortran/radlw/dump"
+serializer = ser.Serializer(ser.OpenModeKind.Read, ddir, "Serialized_rank0")
+savepoints = serializer.savepoint_list()
+
+rebuild = False
+validate = True
+backend = "gtc:gt:cpu_ifirst"
+
+invars = [
+    "totuflux",
+    "totuclfl",
+    "totdflux",
+    "totdclfl",
+    "htr",
+    "htrcl",
+]
+
+outvars = [
+    "upfxc_t",
+    "upfx0_t",
+    "upfxc_s",
+    "upfx0_s",
+    "dnfxc_s",
+    "dnfx0_s",
+    "hlwc",
+    "hlw0",
+]
+
+indict = dict()
+indict_gt4py = dict()
+
+for var in invars:
+    tmp = serializer.read(var, serializer.savepoint["lwrad-finalloop-input-000000"])
+    if var == "htr" or var == "htrcl":
+        tmp2 = np.insert(tmp, 0, 0, axis=1)
+        indict[var] = np.tile(tmp2[:, None, :], (1, 1, 1))
+    else:
+        indict[var] = np.tile(tmp[:, None, :], (1, 1, 1))
+
+    indict_gt4py[var] = create_storage_from_array(
+        indict[var], backend, shape_nlp1, DTYPE_FLT
+    )
+
+outdict_gt4py = dict()
+for var in outvars:
+    if var == "hlwc" or var == "hlw0":
+        outdict_gt4py[var] = create_storage_zeros(backend, shape_nlp1, DTYPE_FLT)
+    else:
+        outdict_gt4py[var] = create_storage_zeros(backend, shape_2D, DTYPE_FLT)
+
+lhlw0 = True
+
+
+@stencil(backend=backend, rebuild=rebuild, externals={"lhlw0": lhlw0})
+def finalloop(
```

Comment:
is there another name that could be more physically descriptive? It looks like it's copying variables -- does this copy need to happen? I know the fortran likes to do that, but sometimes you can just output the data arrays rather than copy them. Could this stencil be merged with the step before?

Reply:
Good point, I'll look at it

Reply:
I've now incorporated this into the previous stencil